test/integration/options.spec.js

```diff
@@ -91,6 +93,29 @@ describe('options', function () {
     });
   });
 
+  describe('--file', function () {
```

Comment:
can we also test multiple `--file` flag uses?  this is basically a copy/paste of the test on L101 w/o the first parameter to `runMochaJSON`; instead, appending to `args`.  

you *may* need to monkey with `runMochaJSON` to optionally accept an array as the first argument...

Reply:
Added an individual additional test that should provide coverage for this.